server/users.ts

```diff
@@ -1676,7 +1676,8 @@ function socketReceive(worker: StreamWorker, workerid: number, socketid: string,
 	}
 	const deltaTime = Date.now() - startTime;
 	if (deltaTime > 1000) {
-		Monitor.warn(`[slow] ${deltaTime}ms - ${user.name} <${connection.ip}>: ${roomId}|${message}`);
+		const logRoom = Rooms.get('slowlog');
+		(logRoom ? logRoom.add : Monitor.warn)(`[slow] ${deltaTime}ms - ${user.name} <${connection.ip}>: ${roomId}|${message}`);
```

Comment:
Why do you have one log in `chat` and one log in `users`? They should both be in one place, shouldn't they?

Reply:
lol i just said that in DMs 
ill move it to chat as well

Reply:
Putting them both in `users` is generally better, because they're both in `users`, but we might need to be better at propagating the promises.